services/cases-api/lambdas/createCase.js

```diff
@@ -39,13 +39,15 @@ export async function main(event) {
   const PK = `USER#${personalNumber}`;
   const SK = `USER#${personalNumber}#CASE#${id}`;
   const timestamp = Date.now();
+  const expirationTime = generateExpirationTime(72);
```

Comment:
This input of number 72 should be moved to a common place, since we will use this in other services as discussed. If you prefer to move this now or later is up to you @dannil76